playwright-e2e/tests/dsm/participant-list/download-all-data.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { expect } from '@playwright/test';
+import { StudyEnum } from 'dsm/component/navigation/enums/selectStudyNav-enum';
+import ParticipantListPage from 'dsm/pages/participant-list-page';
+import { test } from 'fixtures/dsm-fixture';
+import { assertParticipantListDownloadFileName } from 'utils/test-utils';
+
+test.describe.parallel('Participant List Download', () => {
+  // Chose studies with fewer participants on Dev and Test. Otherwise download all data will take a very long time.
+  const studies = [StudyEnum.BRUGADA, StudyEnum.LMS, StudyEnum.OSTEO];
+
+  for (const study of studies) {
+    test(`Select All in ${study} @dsm @${study}`, async ({ page, request }) => {
+      const participantListPage = await ParticipantListPage.goto(page, study, request);
+
+      // Download with “Select all” selected
+      await participantListPage.selectAll();
+
+      // Export as human-readable, Excel and include all completion of an activity
+      const download = await participantListPage.downloadParticipantList();
+      assertParticipantListDownloadFileName(download, study);
+
+      const downloadedFile = await download.path();
+      expect(downloadedFile).toBeTruthy();
+
+      // TODO unzip and check xlsx contents
```

Comment:
Download file is a `.zip` file which contains the `.xlsx` file. In order to read the zip file, I need to install a new third-party lib.